cmd/av/stack_branch.go

```diff
@@ -36,13 +37,17 @@ given as the first argument to the command. Branches should only be renamed
 with this command (not with git branch -m ...) because av needs to update
 internal tracking metadata that defines the order of branches within a stack.`,
 	SilenceUsage: true,
-	RunE: func(cmd *cobra.Command, args []string) (reterr error) {
+	Args: func(cmd *cobra.Command, args []string) error {
 		if len(args) != 1 {
-			_ = cmd.Usage()
-			return errors.New("exactly one branch name is required")
+			return errors.Errorf("accepts 1 arg, received %d", len(args))
 		}
-		branchName := args[0]
-
+		c := strings.Count(args[0], ":")
+		if c > 1 {
+			return errors.New("the branch name should be NEW_BRANCH or OLD_BRANCH:NEW_BRANCH")
```

Comment:
I think this check should just be inside the `stackBranchMove` function.

Reply:
Done.